docs/en/changes/changes.md

```diff
@@ -52,6 +53,7 @@
 * Fix the title of instance list and notices in the continue profiling.
 * Add a link to explain the expression metric, add units in the continue profiling widget.
 * Calculate string width to set Tabs name width.
+* Removed '&' symbols from shell scripts to avoid web application server process running as a background process.
```

Comment:
```suggestion
* [Breaking change] Removed '&' symbols from shell scripts to avoid web application server process running as a background process.
```